src/routes/forms/[uuid]/index.svelte

```diff
@@ -53,7 +73,7 @@
 		{/if}
 	{/each}
 
-	<button class="btn btn-primary indent" on:click={handleSubmit}>Save and continue</button>
+	<button class="btn btn-primary indent" on:click={handleSubmit}>Submit</button>
```

Comment:
I'd like to hear more about this – I'd really liked the idea of "save and continue." Is that not going to work?

Reply:
My assumption was that the "save and continue" button should enter a user-side review state without inserting any nodes into the database, and that the user would have one final "confirm/submit" button in that review state to finally process the form (which is what I intended this button to be) - did I misunderstand the idea behind that?

Reply:
```suggestion
	<button class="btn btn-primary indent" on:click={handleSubmit}>Continue</button>
```